pkg/kn/commands/plugin/handler_test.go

```diff
@@ -90,7 +90,7 @@ func TestPluginHandler(t *testing.T) {
 		})
 
 		t.Run("when plugin is in $PATH", func(t *testing.T) {
-			t.Run("--lookup-plugins-in-path=true", func(t *testing.T) {
+			t.Run("--lookup-plugins=true", func(t *testing.T) {
```

Comment:
Isn't it that we now have `--lookup-plugins` and `--no-lookup-plugins` for boolean params ?